cli/2024-04-azure-workload-identity.md

```diff
@@ -0,0 +1,233 @@
+# Radius Azure Workload Identity Support
+
+* **Status**: In Review
+* **Author**: Will Smith (@willdavsmith)
+
+## Overview
+
+A software workload such as a container-based application, service or script needs an identity to authenticate, access, and communicate with services that are distributed across different platforms and/or cloud providers. For workloads running outside of Azure, the workload need a credential (secret or certificate) to access Azure resources. These credentials pose a security threat and must be stored and rotated regularly.  
+
+Radius supports service principal identity to configure the Azure provider to deploy and manage Azure resources. However, this adds an extra task for operators to rotate the credentials for the identity. A workload identity or federated identity helps us avoid this maintenance challenge of manually managing the credentials and eliminates the risk of exposing secrets or having certificates expire. 
+
+The goal of the scenario is to enable infrastructure operators to configure workload identity support for the Azure provider in Radius to deploy and manage Azure resources.
+
+## Terms and definitions
+
```

Comment:
Can you add something here making it clear these terms are defined by Azure/Entra ID. They may not have meanings, or may have different meanings outside of that context.